crates/bevy_render/src/color/colorspace.rs

```diff
@@ -102,6 +102,123 @@ impl HslRepresentation {
     }
 }
 
+pub struct LchRepresentation;
+impl LchRepresentation {
+    const CIE_EPSILON: f32 = 216.0 / 24389.0;
```

Comment:
All of these magic numbers need to be sourced. A single good reference is fine, but ideal would be specific callouts.